dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -223,215 +140,131 @@ private static Link getLinkFields(Map<String, String> map) throws ParseException
      * @param results
      * @return
      */
-    public static List<Identifier> convertDotConnectMapToIdentifier(final List<Map<String,String>> results)
+    public static List<Identifier> convertDotConnectMapToIdentifier(final List<Map<String, Object>> results)
             throws ParseException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.text.ParseException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)